coreos-postinst

```diff
@@ -127,6 +127,19 @@ if [ -e ${PLATFORM_PATH} ]; then
     fi
 fi
 
+# Our VMware OEM partition contained a version of vmtoolsd that was vulnerable to 
+# CVE-2015-5191 (http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-5191).
+# This CVE can be mitigated by setting PrivateTmp=true on the vmtoolsd service file.
+# coreos-base/oem-vmware-10.1.5 is the last vulnerable version of the oem ebuild.
+VMTOOLSD_DROPIN=/etc/systemd/system/vmtoolsd.service.d/90-tmpfiles-cve-2015-4191.conf
+if [ ! -f $VMTOOLSD_DROPIN ] && grep -q coreos.oem.id=vmware /proc/cmdline; then
```

Comment:
Old versions of the VMware OEM don't set `coreos.oem.id`, even after upgrades.  (Tested with 735.0.0.)  I think the correct approach is to also check `/etc/oem-release`; @crawford?